.changeset/forty-radios-try.md

```diff
@@ -0,0 +1,7 @@
+---
+"@pnpm/get-context": patch
```

Comment:
for `@pnpm/get-context` this is  actually a feature. So I guess we need a separate changeset for it, telling about the new unhookedManifest

Reply:
changesets updated